### PR TITLE
add ipfs developer relations

### PR DESCRIPTION
**Pull request by 2color**
### Summary
Add IPFS developer relations team

### CODE REVIEW

File: github/ipfs.yml [hunk 1]
```diff
@@ -5686,6 +5686,18 @@ teams:
         - ntninja
         - whereswaldon
     privacy: closed
+  ipfs-devrel:
```
Comment (galargh) [inline github/ipfs.yml:5689]: If we go with just devrel, then we could mention it with @ipfs/devrel
```suggestion
 devrel:
```

Completely up to you, let me know :)

**Review by galargh — APPROVED** [review]

File: github/ipfs.yml (hunk 1 above)
Comment (2color) [reply github/ipfs.yml]: much better :)

**Comment by 2color** [comment]
@galargh Do you know why the build failed?

File: github/ipfs.yml [hunk 2]
```diff
@@ -5686,6 +5686,18 @@ teams:
         - ntninja
         - whereswaldon
     privacy: closed
+ devrel:
```
Comment (galargh) [inline github/ipfs.yml:5689]:
```suggestion
  devrel:
```

**Comment by galargh** [comment]
I didn't put enough spaces in front of `devrel` 🤦

**Pull request merged**